docs/reference/migration/migrate_7_0/mappings.asciidoc

```diff
@@ -5,6 +5,11 @@
 
 The `_all` field deprecated in 6 have now been removed.
 
+==== The `_default_` mapping is disallowed
+
+The `_default_` mapping has been deprecated in 6.0 and is now disallowed in 7.0.
```

Comment:
minor: Instead of `disallowed`  could we say `no longer allowed`? To me it reads a bit easier